photo-gallery/src/imageItem.js

```diff
@@ -104,15 +104,14 @@ const hornedBeastData = [
       horns: 2
```

Comment:
ImageItem should be a component rendering what you want each individual animal card to look like.